_rules/zoom-text-no-overflow-clipping-59br37.md

```diff
@@ -0,0 +1,247 @@
+---
+id: 59br37
+name: Zoomed text node is not clipped with CSS overflow
+rule_type: atomic
+description: |
+  This rule checks that text nodes are not unintentionally clipped by overflow, when a page is zoomed to 200% on 1280 by 1024 display;
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:1.4.4: # Resize Text (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Wilco Fiers
+  assets:
+    - The Raven, poem by Edgar Allan Poe
+---
+
+## Applicability
+
+The rule applies to any [text node][] for which all of the following is true when in a [display size][] of 640 by 512:
+
+- The [text node][] is [visible][]; and
+- The [text node][] has an HTML element as a [parent][] in the [flat tree][]; and
+- The [text node][] has an [ancestor][] in the [flat tree][] with the a [computed][] [overflow-x][overflow] or [overflow-y][overflow] of `none` or `clip`; and
```

Comment:
Does the `getComputedStyle(node).overflow` return `clip` or `none`? The `text-overflow` does return `clip`, but not `overflow`.

Reply:
`overflow-{x,y}` computes as follows:

> as specified, except with `visible`/`clip` computing to `auto`/`hidden` (respectively) if one of `overflow-x` or `overflow-y` is neither `visible` nor `clip`.

https://drafts.csswg.org/css-overflow-3/#propdef-overflow-x

`none` isn't a valid value for `overflow-{x,y}` so that should probably be fixed though.

Reply:
good catch. That obviously should have been "hidden" instead of "none.